### ISSUE TITLE
FBX Importer Error (Ubuntu 20.04, arm64)

### ISSUE THREAD

**Issue by lilrokstarr** [MonoGame/MonoGame#7364]
I am currently running **.NET SDK version 3.1.102 _arm64_** on **Ubuntu20.04.**
The SDK is installed in the home directory.

I have all the necessary dependecies, including **LibFreeImage-Dev** and **libassimp-dev**.

However, when I proceed to build my content, the FBX Importer reads: _Arithmetic Operation resulted in an overflow._

Assuming the issue was with **ARM64 Linux**, I proceeded to build my content using an **AMD64 machine running Windows 10**, and then re-imported the successful build on to my **ARM64** machine.

However, upon attempting to run my project, _dotnet CLI_ spit out the following error:
```
/home/rock/Dev/Monogame/3D/Content/3Dobjecs/cone.fbx : error : Importer 'FbxImporter' had unexpected failure! [/home/rock/Dev/Monogame/3D/3D.csproj]
/home/rock/Dev/Monogame/3D/Content/3Dobjecs/cube.fbx : error : Importer 'FbxImporter' had unexpected failure! [/home/rock/Dev/Monogame/3D/3D.csproj]
/home/rock/Dev/Monogame/3D/Content/3Dobjecs/cylinder.fbx : error : Importer 'FbxImporter' had unexpected failure! [/home/rock/Dev/Monogame/3D/3D.csproj]
/home/rock/Dev/Monogame/3D/Content/3Dobjecs/donut.fbx : error : Importer 'FbxImporter' had unexpected failure! [/home/rock/Dev/Monogame/3D/3D.csproj]
/home/rock/Dev/Monogame/3D/Content/3Dobjecs/pill.fbx : error : Importer 'FbxImporter' had unexpected failure! [/home/rock/Dev/Monogame/3D/3D.csproj]
/home/rock/Dev/Monogame/3D/Content/3Dobjecs/plane.fbx : error : Importer 'FbxImporter' had unexpected failure! [/home/rock/Dev/Monogame/3D/3D.csproj]
/home/rock/Dev/Monogame/3D/Content/3Dobjecs/sphere.fbx : error : Importer 'FbxImporter' had unexpected failure! [/home/rock/Dev/Monogame/3D/3D.csproj]
/home/rock/.nuget/packages/monogame.content.builder.task/3.8.0.1641/build/MonoGame.Content.Builder.Task.targets(138,5): error MSB3073: The command "dotnet /home/rock/.nuget/packages/monogame.content.builder.task/3.8.0.1641/build/../tools/netcoreapp3.1/any/mgcb.dll /quiet /@:"/home/rock/Dev/Monogame/3D/Content/Content.mgcb" /platform:DesktopGL /outputDir:"/home/rock/Dev/Monogame/3D/Content/bin/DesktopGL/Content" /intermediateDir:"/home/rock/Dev/Monogame/3D/Content/obj/DesktopGL/Content" /workingDir:"/home/rock/Dev/Monogame/3D/Content/"" exited with code 7. [/home/rock/Dev/Monogame/3D/3D.csproj]

The build failed. Fix the build errors and run again.
```

#### What version of MonoGame does the bug occur on:
- MonoGame 3.8.0.1641

#### What operating system are you using:
- Ubuntu (ARM64)

#### What MonoGame platform are you using:
- DesktopGL

**Comment by rejurime** [comment]
Hi!
In Ubuntu 20.04 I had problems importing models with textures: https://community.monogame.net/t/problem-loading-models-with-textures-in-ubuntu-20-04-with-mg-3-8/
I think the fix to this problem comes out in version 3.8.1 with this pr https://github.com/MonoGame/MonoGame/pull/7354

**Comment by harry-cpp** [comment]
We currently don't support content building from arm machines, this will probably change down the line.
